src/components/Navbar/Navbar.component.jsx

```diff
@@ -0,0 +1,239 @@
+import React from 'react';
+import clsx from 'clsx'
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import IconButton from '@material-ui/core/IconButton';
+import InputBase from '@material-ui/core/InputBase';
+import MenuItem from '@material-ui/core/MenuItem';
+import Menu from '@material-ui/core/Menu';
+import MenuIcon from '@material-ui/icons/Menu';
+import SearchIcon from '@material-ui/icons/Search';
+import AccountCircle from '@material-ui/icons/AccountCircle';
+import MoreIcon from '@material-ui/icons/MoreVert';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Switch from '@material-ui/core/Switch';
+import { withStyles } from '@material-ui/core/styles';
+// import { createMuiTheme } from '@material-ui/core/styles';
+import { useTheme } from '@material-ui/core/styles';
+import Divider from '@material-ui/core/Divider';
+import ChevronLeftIcon from '@material-ui/icons/ChevronLeft';
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import HomeIcon from '@material-ui/icons/Home';
+import List from '@material-ui/core/List';
+import Drawer from '@material-ui/core/Drawer';
+
+
+import useStyles from './styles';
+
+
+export default function Navbar() {
+  const classes = useStyles();
+  const theme = useTheme();
+  const [anchorEl, setAnchorEl] = React.useState(null);
+  const [mobileMoreAnchorEl, setMobileMoreAnchorEl] = React.useState(null);
+
+  const isMenuOpen = Boolean(anchorEl);
+  const isMobileMenuOpen = Boolean(mobileMoreAnchorEl);
+
+  // Burger menu ///////////////////////
+  const [open, setOpen] = React.useState(false);
```

Comment:
The comments are good, sometimes are necessary.

On this case, you can make self explained the code renaming the variables, for example: `isBurgerMenuOpen` and `setIsBurgerMenuOpen`, with that, you can avoid the comment because the variable names indicates what is the reason of their existence.

The same apply for other types of names: functions, components, classes, files, etc